config.c

```diff
@@ -1101,6 +1101,12 @@ static int readConfigFile(const char *configFile, struct logInfo *defConfig)
                         newlog->flags |= LOG_FLAG_DATEYESTERDAY;
                     } else if (!strcmp(key, "datehourago")) {
                         newlog->flags |= LOG_FLAG_DATEHOURAGO;
+#if defined(WITH_FALLOCATE)
+                    } else if (!strcmp(key, "copyreduce")) {
+                        newlog->flags |= LOG_FLAG_COPYREDUCE;
+                    } else if (!strcmp(key, "nocopyreduce")) {
+                        newlog->flags &= ~LOG_FLAG_COPYREDUCE;
+#endif
```

Comment:
maybe use a different error message if used on an unsupported platform than the generic message?
e.g.:
```c
#else
    } else if (!strcmp(key, "copyreduce") || !strcmp(key, "nocopyreduce")) {
        message(MESS_ERROR, "%s:%d unknown option '%s' "
                "-- ignoring line\n", configFile, lineNum, key);
        if (*start != '\n')
            state = STATE_SKIP_LINE;
#endif
```

Reply:
`unknown option` sounds clear to me.  I would not over-complicate this code, which is nearly unmaintanable already.

Reply:
"unsupported option" looks good for me.